### PR TITLE
support for CPU training

### PR DESCRIPTION
**Pull request by wangruohui**
I am not quite sure whether this is a bug in historical codes. Because it seems currently no package support training with CPU so the modified line will never enter.

### CODE REVIEW

**Comment by yptheangel** [comment]
do you support CPU inference?

**Comment by wangruohui** [comment]
> do you support CPU inference?

It has been supported in master. And this PR does not break this functionality.

**Comment by wangruohui** [comment]
> do you support CPU inference?

It has been supported in master. And this PR does not break this functionality.

**Review by zhouzaida — APPROVED** [review]
LGTM

File: mmcv/parallel/data_parallel.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def train_step(self, *inputs, **kwargs):
             # We add the following line thus the module could gather and
             # convert data containers as those in GPU inference
             inputs, kwargs = self.scatter(inputs, kwargs, [-1])
-            return self.module.train_step(*inputs, **kwargs)
+            return self.module.train_step(*inputs[0], **kwargs[0])
```
Comment (ZwwWayne) [inline mmcv/parallel/data_parallel.py:52]: Shall we also modify that in val_step?

**Pull request merged**